doc/TOOLBOX.md

```diff
@@ -0,0 +1,52 @@
+# Toolbox testing guide
+
+## Before You Run It
+
+### Requirements
+
+* RHEL 7/8 machine
+* docker/podman installed on RHEL machine
+* sshd installed and configured on RHEL machine
+* Toolbox image pulled on RHEL machine
+* two 3scale tenants adn related access tokens
+
+### Configuration
+
+Extra configuration option for Toolbox tests is needed. This is example of configuration:
+
+```yaml
+toolbox:
+  toolbox:
+    cmd: "podman"
+    podman_image: "" # hash of Toolbox image from `podman image list`
+    destination_endpoint: "" # url to 3scale API endpoint of second tenant
+    destination_provider_key: "" # access token for `destination_endpoint`
+    machine_ip: "" # RHEL machine IP
+    ssh_user: "" # SSH user for access to RHEL machine who is able to run podman images
+    ssh_passwd: "" # SSH password of `ssh_user`
+
+  threescale:
```

Comment:
Is this needed even if you are logged in to openshift?

Reply:
It is not needed. I would like to just stress it out. So it is clear that Toolbox uses also these options. I am going to add there some text to clear it more.